src/NHSD.BuyingCatalogue.Identity.UserDatabase/Deployment/CreateTestBuyers.sql

```diff
@@ -0,0 +1,40 @@
+    (N'b12055c4-5960-414a-b0d9-389d515ef5d9', N'Hilario.Bauch93@gmail.com', N'HILARIO.BAUCH93@GMAIL.COM', N'Hilario.Bauch93@gmail.com', N'HILARIO.BAUCH93@GMAIL.COM', 0, NULL, N'716298b6-ec16-4a3e-a46f-fbab15273b00', N'ff9d9d29-27a0-4c5c-b7c6-4988ce0808e9', N'49730020308', 0, 0, NULL, 0, 0, N'145cdd34-0bb9-4122-a0bf-1b6fcb0219bb', N'Buyer', 0, 0, N'Hilario', N'Bauch'),
+    (N'b6275809-20df-4b86-9458-4bee85566246', N'Alvina7@gmail.com', N'ALVINA7@GMAIL.COM', N'Alvina7@gmail.com', N'ALVINA7@GMAIL.COM', 0, NULL, N'cf5e715f-61e2-4717-8fd7-79ca64855f9c', N'ffa05f0f-7943-4eb1-add4-acac2b07bfbf', N'52331650575', 0, 0, NULL, 0, 0, N'd795a598-26f0-46fb-8a6e-f22cd912d7dc', N'Buyer', 0, 0, N'Alvina', N'Boyle'),
+    (N'b9a3e0de-849c-4301-b9e1-41a6c26c7b9b', N'Ransom.Nolan9@gmail.com', N'RANSOM.NOLAN9@GMAIL.COM', N'Ransom.Nolan9@gmail.com', N'RANSOM.NOLAN9@GMAIL.COM', 0, NULL, N'7e4f0c77-a4fc-43c4-8fb8-97ab75fa7928', N'61547d0a-6463-4121-80ba-626fed63a24b', N'85512460808', 0, 0, NULL, 0, 0, N'e99efe1f-273a-4de8-9b18-c466c15d7560', N'Buyer', 0, 0, N'Ransom', N'Nolan'),
+    (N'c80d3080-0684-4ace-925f-5a62d90107b6', N'Laura_Emmerich57@gmail.com', N'LAURA_EMMERICH57@GMAIL.COM', N'Laura_Emmerich57@gmail.com', N'LAURA_EMMERICH57@GMAIL.COM', 0, NULL, N'feea77a2-c501-488c-abf2-f484e86d67f3', N'83748796-2d46-4411-bbd2-618c818ea46d', N'83201713079', 0, 0, NULL, 0, 0, N'bb21dd07-58df-457a-a9cb-6b035684692b', N'Buyer', 0, 0, N'Laura', N'Emmerich'),
+    (N'cc69e5a6-a35f-4e33-94ec-3be39fe81423', N'Evans.Friesen49@yahoo.com', N'EVANS.FRIESEN49@YAHOO.COM', N'Evans.Friesen49@yahoo.com', N'EVANS.FRIESEN49@YAHOO.COM', 0, NULL, N'1078ba5b-c9f3-4918-8dff-025b6f0cee80', N'745d5a99-1a0a-4eab-9ba3-85166d696cee', N'21496157737', 0, 0, NULL, 0, 0, N'bb21dd07-58df-457a-a9cb-6b035684692b', N'Buyer', 0, 0, N'Evans', N'Friesen'),
+    (N'd94109ef-6051-4937-8a9e-8599a2707096', N'Larry.Ebert94@gmail.com', N'LARRY.EBERT94@GMAIL.COM', N'Larry.Ebert94@gmail.com', N'LARRY.EBERT94@GMAIL.COM', 0, NULL, N'49bdba08-357a-4204-b3a1-25c73f582ca6', N'146602f2-3090-4543-8edc-3d078dd470e2', N'95740022454', 0, 0, NULL, 0, 0, N'b1e85a4d-364c-4adc-a981-eb21b58a2cc0', N'Buyer', 0, 0, N'Larry', N'Ebert'),
+    (N'db70b19d-5387-4d36-8957-a64dd9d87e66', N'Grover29@yahoo.com', N'GROVER29@YAHOO.COM', N'Grover29@yahoo.com', N'GROVER29@YAHOO.COM', 0, NULL, N'c2c9ca3f-597a-44a8-bc47-75433cf89cdb', N'606403ae-ac27-4a57-bb6f-bd73605357e2', N'74529061674', 0, 0, NULL, 0, 0, N'145cdd34-0bb9-4122-a0bf-1b6fcb0219bb', N'Buyer', 0, 0, N'Grover', N'Nader'),
+    (N'e8fd68d0-5ec0-40d3-a2b1-58cd4769b099', N'Santos33@gmail.com', N'SANTOS33@GMAIL.COM', N'Santos33@gmail.com', N'SANTOS33@GMAIL.COM', 0, NULL, N'c8a655f9-4a64-48d3-b0ea-1b3e07604435', N'fb0ec71c-e961-44a5-9b7e-a8a4beb62b85', N'41140608953', 0, 0, NULL, 0, 0, N'd795a598-26f0-46fb-8a6e-f22cd912d7dc', N'Buyer', 0, 0, N'Santos', N'Kuhlman')
```

Comment:
I'm wondering if we probably ought to use reserved DNS names for our test users to minimize the risk of actually sending an e-mail to a genuine address. These are domains that a real SMTP server should not resolve – see [RFC 2606](https://tools.ietf.org/html/rfc2606#page-2) for more details.

The reserved names are:

- .test
- .example
- .invalid
- .localhost
- example.com
- example.net
- example.org